src/findings_data_import.py

```diff
@@ -49,12 +49,288 @@ def move_processed_file(s3_client, bucket, folder, filename):
+
+    Parameters
+    ----------
+    db_hostname : str
+        The hostname that has the database to store the data in.
+
+    db_port : str
+        The port that the database server is listening on. [default: 27017]
+
+    ssm_db_name : str
+        The name of the parameter in AWS SSM that holds the name of the
+        database to store the assessment data in.
+
+    ssm_db_user : str
+        The name of the parameter in AWS SSM that holds the database username
+        with write permission to the assessment database.
+
+    ssm_db_password : str
+        The name of the parameter in AWS SSM that holds the database password
+        for the user with write permission to the assessment database.
+
+    Returns
+    -------
+    database : A database object returned from the MongoClient
+    """
+    try:
+        logging.info("Grabbing database credentials from SSM.")
+        # Fetch database credentials from AWS SSM
+        ssm_client = boto3_client("ssm")
+
+        db_info = dict()
+        for ssm_param_name, key in (
+            (ssm_db_name, "db_name"),
+            (ssm_db_user, "username"),
+            (ssm_db_password, "password"),
+        ):
+            response = ssm_client.get_parameter(
+                Name=ssm_param_name, WithDecryption=True
+            )
+            db_info[key] = response["Parameter"]["Value"]
+
+        logging.info(f"Connecting to the mongo db at {db_hostname} {db_port}")
+        # Set up database connection
+        credPw = urllib.parse.quote(db_info["password"])
+        db_uri = (
+            f"mongodb://{db_info['username']}:{credPw}@"
+            f"{db_hostname}:{db_port}/{db_info['db_name']}"
+        )
+
+        # Connect to MongoDB with timeout so Lambda doesn't run over
+        db_connection = MongoClient(
+            host=db_uri, serverSelectionTimeoutMS=2500, tz_aware=True
+        )
+        db = db_connection[db_info["db_name"]]
+        logging.info(
+            f"DB connection set up to {db_hostname}:{db_port}/{db_info['db_name']}"
+        )
+        return db
+    except ClientError as client_err:
+        raise Exception(
+            "Unable to fetch database credentials from the SSM", client_err
+        ).with_traceback()
+    # Handle all mongo exceptions the same way..
+    except Exception as err:
+        raise Exception("Unable to connect to the mongo db", err).with_traceback()
+
+
+def download_file(
+    s3_client=None,
+    s3_bucket=None,
+    data_filename=None,
+):
+    """Download a file from a specified s3 bucket, and place it in a temporary file path.
+
+    Parameters:
+    -----------
+    s3_client : S3.Client
+        The AWS S3 Client to retrieve the data file with .
+
+    s3_bucket : str
+        The AWS S3 bucket containing the data file.
+
+    data_filename : str
+        The name of the file containing the data in the S3 bucket
+        above.
+
+    Returns
+    -------
+        string, : The file path of the newly created temp file
```

Comment:
I think there is an extra comma here:
```suggestion
        string : The file path of the newly created temp file
```